<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePassword</class>
 <widget class="QMainWindow" name="ChangePassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改密码</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="oldlabel">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>旧密码：   </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="oldEdit">
         <property name="styleSheet">
          <string notr="true">QDialog {
	background: #D6DBE9;
}

QLineEdit {
	border: 2px solid #000000; /* 边框宽度为1px，颜色为#A0A0A0 */
	border-radius: 10px; /* 边框圆角 */
	padding-left: 5px; /* 文本距离左边界有5px */
	background-color: 	#FFFFFF	; /* 背景颜色 */
	color: 	#000000; /* 文本颜色 */
	selection-background-color: #000000; /* 选中文本的背景颜色 */
	selection-color: #FFFFFF; /* 选中文本的颜色 */
	font-family: &quot;Microsoft YaHei&quot;; /* 文本字体族 */
	font-size: 10pt; /* 文本字体大小 */
}

QLineEdit:hover { /* 鼠标悬浮在QLineEdit时的状态 */
	border: 2px solid #298DFF;
	border-radius: 10px;
	background-color: #FFFFFF;
	color: 	#000000;
	selection-background-color: #000000;
	selection-color: #FFFFFF;
}


QLineEdit:disabled { /* QLineEdit在禁用时的状态 */
	border: 2px solid 	#C0C0C0;
	background-color: 	#C0C0C0;
	color: 	#000000;
}

QLineEdit:read-only { /* QLineEdit在只读时的状态 */
	border: 2px solid 	#C0C0C0;
	background-color: 	#C0C0C0;
	color: 	#000000;
}</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="newlabel">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>新密码：   </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="newEdit">
         <property name="styleSheet">
          <string notr="true">QDialog {
	background: #D6DBE9;
}

QLineEdit {
	border: 2px solid #000000; /* 边框宽度为1px，颜色为#A0A0A0 */
	border-radius: 10px; /* 边框圆角 */
	padding-left: 5px; /* 文本距离左边界有5px */
	background-color: 	#FFFFFF	; /* 背景颜色 */
	color: 	#000000; /* 文本颜色 */
	selection-background-color: #000000; /* 选中文本的背景颜色 */
	selection-color: #FFFFFF; /* 选中文本的颜色 */
	font-family: &quot;Microsoft YaHei&quot;; /* 文本字体族 */
	font-size: 10pt; /* 文本字体大小 */
}

QLineEdit:hover { /* 鼠标悬浮在QLineEdit时的状态 */
	border: 2px solid #298DFF;
	border-radius: 10px;
	background-color: #FFFFFF;
	color: 	#000000;
	selection-background-color: #000000;
	selection-color: #FFFFFF;
}


QLineEdit:disabled { /* QLineEdit在禁用时的状态 */
	border: 2px solid 	#C0C0C0;
	background-color: 	#C0C0C0;
	color: 	#000000;
}

QLineEdit:read-only { /* QLineEdit在只读时的状态 */
	border: 2px solid 	#C0C0C0;
	background-color: 	#C0C0C0;
	color: 	#000000;
}</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="new2label">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>确认密码：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="new2Edit">
         <property name="styleSheet">
          <string notr="true">QDialog {
	background: #D6DBE9;
}

QLineEdit {
	border: 2px solid #000000; /* 边框宽度为1px，颜色为#A0A0A0 */
	border-radius: 10px; /* 边框圆角 */
	padding-left: 5px; /* 文本距离左边界有5px */
	background-color: 	#FFFFFF	; /* 背景颜色 */
	color: 	#000000; /* 文本颜色 */
	selection-background-color: #000000; /* 选中文本的背景颜色 */
	selection-color: #FFFFFF; /* 选中文本的颜色 */
	font-family: &quot;Microsoft YaHei&quot;; /* 文本字体族 */
	font-size: 10pt; /* 文本字体大小 */
}

QLineEdit:hover { /* 鼠标悬浮在QLineEdit时的状态 */
	border: 2px solid #298DFF;
	border-radius: 10px;
	background-color: #FFFFFF;
	color: 	#000000;
	selection-background-color: #000000;
	selection-color: #FFFFFF;
}


QLineEdit:disabled { /* QLineEdit在禁用时的状态 */
	border: 2px solid 	#C0C0C0;
	background-color: 	#C0C0C0;
	color: 	#000000;
}

QLineEdit:read-only { /* QLineEdit在只读时的状态 */
	border: 2px solid 	#C0C0C0;
	background-color: 	#C0C0C0;
	color: 	#000000;
}</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_4" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="returnButton">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	font: 12pt &quot;幼圆&quot;;
    color: 	#EE82EE;
    background-color: #FFFFFF;
    border-color: 	#EE82EE;
    border-radius: 10px;
    border-style: solid;
    border-width: 1px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #FFFFFF;
    background-color: #EE82EE;
    border-color: #EE82EE;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #FFFFFF;
    background-color: 	#DA70D6;
	border-color: 	#DA70D6;
}</string>
         </property>
         <property name="text">
          <string>返回</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="confirmButton">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	font: 12pt &quot;幼圆&quot;;
    color: 	#EE82EE;
    background-color: #FFFFFF;
    border-color: 	#EE82EE;
    border-radius: 10px;
    border-style: solid;
    border-width: 1px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #FFFFFF;
    background-color: #EE82EE;
    border-color: #EE82EE;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #FFFFFF;
    background-color: 	#DA70D6;
	border-color: 	#DA70D6;
}</string>
         </property>
         <property name="text">
          <string>确认</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
